pkgs/tools/typesetting/tex/texlive/build-texlive-package.nix

```diff
@@ -0,0 +1,141 @@
+{ lib
+, fetchurl
+, runCommand
+
+  # script interpreters
+, bash
+, jdk
+, perl
+, python3
+, ruby
+, snobol4
+, tk
+
+  # TeX Live prerequisites
+, texliveBinaries
+, texliveFixedHashes
+, texliveMirrors
+}:
+
+{ pname
```

Comment:
@apfelkuchen6 I am trying to make `buildTeXLivePackage` more idiomatic, so I pulled `pname` into the attribute set, with negligible impact on performance I hope.

I believe we should also move fixed hashes and mirrors down into the attribute set somehow, but that's more complicated and may require improvements to `fixed-hashes.nix` – I'd rather do this later, to keep the review easy.